packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -157,7 +157,7 @@ const rewriteDeprecatedAppNames = (
 ): RuntimeConfiguration => {
   const appAliases = [
     { name: 'preview', oldName: 'media-viewer' },
-    { name: 'simple-editor', oldName: 'markdown-editor' }
+    { name: 'text-editor', oldName: 'markdown-editor' }
```

Comment:
Could have been handy to make oldName an array of strings to check against multiple old aliases but not relevant for not I'd say

Reply:
since it's not exposed we can always do so if we need it. prefer to keep it as simple as possible for now. :-)